### DISCUSSION TITLE
Paused downloads getting cleared why?

### DISCUSSION THREAD

**Discussion by DemonoidMaster** [TeamNewPipe/NewPipe#10569] [Questions]
So if there is a download and the user decides to pause it, why the fuck does the app auto-deletes what's paused from the downloads list if the app is closed OR refreshed due to low ram etc? It would be pretty nice if it DIDN'T do that, causing the user to have to redownload the entire video FROM SCRATCH.

Also uh when will those 1.2k open bug reports get fixed? Seems to me like at this point it's better to just use Youtube and move on from using newpipe if you just abandoned it causing anyother fork to be stalled too because you can't be bothered to just give the reigns to somebody else.

**Comment by opusforlife2** [comment]
> Also uh when will those 1.2k open bug reports get fixed? 

You're welcome to help out. You can start with the ones labelled `good first issue`. Chop chop. Get to it.

The rest has been answered in issues. Search.
